pod.j2

```diff
@@ -19,17 +19,24 @@ spec:
       resources:
         requests:
           cpu: 16
+          memory: 2G
         limits:
           cpu: 16
+          memory: 2G
       command: ["cyclictest"]
       args:
-        - --smp
         - --duration={{ duration }}
         - --histofall=US
         - --mlockall
-        - --priority=80
-        - --interval=200
+        - --priority=95
+        - --interval=1000
         - --distance=0
+        - --smi
+        - -a
+        - -t 16
+        - -n
+        - -h 30
+        - --refresh_on_max
```

Comment:
This won't generate the histogram dump, are you sure?

Reply:
Well after I tested it with 2G memory & priority=95 & interval=1000 we get the dump